web-app/js/projectActivityInfo.js

```diff
@@ -78,4 +78,14 @@ var pActivityInfo = function(o, selected){
             }
         });
     }
+
+    self.isInfoValid = function () {
+        return self.name() && self.description() && self.startDate() && self.isEndDateAfterStartDate();
+    };
+
+    self.isEndDateAfterStartDate = function () {
+        var start = moment(self.startDate());
+        var end = moment(self.endDate());
+        return end >= start;
```

Comment:
This check doesn't work if the end date is not set: it always results in the error message being displayed.